ablator/main/state.py

```diff
@@ -299,7 +299,7 @@ def sample_trials(self, n_trials_to_sample: int) -> list[ParallelConfig] | None:
             return None
         trials = self.__sample_trials(
             n_trials_to_sample,
-            prev_trials=self.pending_trials,
+            prev_trials=[],
```

Comment:
What is the reasoning for this change?

Reply:
`n_trials_to_sample = min(self.n_trials_remaining, n_trials_to_sample)` this line of code already calculates the number of trials should be sampled, it is safe to ignore these pending trails because we are sampling some new trials. If we let `prev_trials=self.pending_trials` it will cause a bug that the remaining pending trials will be counted as part of the new sampled trials.

Reply:
great find!